examples/swarmion-full-stack/frontend/shared/src/components/NotFound/__tests__/NotFound.test.tsx

```diff
@@ -2,6 +2,13 @@ import { render, screen } from '@testing-library/react';
 
 import NotFound from '../NotFound';
 
+vi.mock('assets', () => ({
+  NotFoundIcon: 'svg',
+  notFoundJpgUrl: 'jpg',
+  notFoundPngUrl: 'png',
+  notFoundWebpUrl: 'webp',
+}));
```

Comment:
Let's go and try to find a way to always mock `*.{svg,jpg,png,webp}` files. It should be done automatically instead of doing it in every test files.